core/Net/DnsRecordReader.php

```
@@ -0,0 +1,245 @@
+        foreach (array_slice($answer, 0, self::MAX_RECORDS_PER_TYPE) as $record) {
+            $line = self::render($record);
+            if ($line !== null) {
+                $values[] = $line;
+            }
+        }
+
+        return [
+            'status' => $values === [] ? self::STATUS_NONE : self::STATUS_FOUND,
+            'values' => $values,
+        ];
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 135. (undefined)

(ErrorControlOperator)